c2corg_api/views/document_associations.py

```diff
@@ -12,23 +13,25 @@
 from c2corg_api.views.document_schemas import waypoint_documents_config, \
     route_documents_config, user_profile_documents_config, \
     image_documents_config, article_documents_config, area_documents_config, \
-    outing_documents_config
+    outing_documents_config, report_documents_config
 from c2corg_api.views.validation import updatable_associations
 from sqlalchemy.sql.expression import or_, and_
 
 associations_to_include = {
     WAYPOINT_TYPE: {
         'waypoints', 'waypoint_children', 'routes', 'images', 'articles'},
     ROUTE_TYPE:
-        {'waypoints', 'routes', 'images', 'articles'},
+        {'waypoints', 'routes', 'images', 'articles', 'reports'},
```

Comment:
@asaunier can you please confirm later if the associations to other document types should be included in the reports? I have found that they existed for route/site/outing/user/article/image here https://github.com/c2corg/camptocamp.org/blob/master/data/sql/tables/db_associations.sql#L52-L63